src/interract.js

```diff
@@ -0,0 +1,19 @@
+// Checkbox function
+export function checkvalue(list) {
+  document.addEventListener('change', (e) => {
+    if (e.target.classList.contains('to-do')) {
+      const index = e.target.dataset.listIndex - 1;
+      e.target.parentNode.children[1].classList.toggle('checked');
+
+      list.checked(index);
+    }
+  });
+}
+
+// Clear all selected tasks
+export function clearAll(list) {
+  const clr = document.querySelector('button');
+  clr.addEventListener('click', () => {
+    list.clear();
+  });
+}
```

Comment:
- [ ] It would be nice if your app should not reload while clearing the completed task 🤔 because right now your app reloads after clearing the completed todos 😨 which is not expected behavior 🤔 It would be nice if you follow the project requirements and your app won't reload while doing the crud operations 👍

**Hint:** Kindly make a display method to display todos after doing crud operations 👍 What this method will do ⬇️
1. First it will clear all take the localStorage and filter the completed items
2. Second, it will remove all the todos from your DOM
3. Third it will render a new list of todos from local storage